manticore/native/cpu/abstractcpu.py

```diff
@@ -11,6 +11,7 @@
 from .disasm import init_disassembler
 from ..memory import ConcretizeMemory, InvalidMemoryAccess, FileMap, AnonMap
 from ..memory import LazySMemory, Memory
+from ..state import State
```

Comment:
Does a cpu needs to know what a state is to work?

Reply:
I understand your concern about separation of concepts. This is only imported to use as a type hint on the `setstate` function, but I can remove this 👍